src/lib/Libauth/munge/munge_supp.c

```diff
@@ -146,35 +146,37 @@ init_munge(void)
  *
  */
 static char *
-munge_get_auth_data(void)
+munge_get_auth_data(char *ebuf, size_t ebufsz)
```

Comment:
Add a comment that ebuf was initialized with nulls all through and ebufsz value passed is sizeof(ebuf) - 1, and that is why we do not need to null terminate the last byte in the snprintf

Reply:
This was actually unneeded.  snprintf() will write as much as N-1 bytes and always null terminate.  It's strncpy() which has the problem.